### PR TITLE
guide for changing provider version

### LINKED ISSUE #206: [Docs]: Create Guide for moving from one Version to another version
### Documentation Link

https://registry.terraform.io/providers/NetApp/netapp-ontap/latest/docs

### Description

We need a guide to tell user what they will need to do if they move from 1.1.1 to 1.1.2 (if there a new field)

### References

_No response_

### Would you like to implement a fix?

None

### PR DESCRIPTION
**Pull request by suhasbshekar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## 1.1.2 (2024-06-03)
 
+ENHANCEMENTS:
+* added guide for changing NetApp Ontap Provider from one version to another.
```
Comment (carchi8py) [inline CHANGELOG.md]: Minor version

File: docs/guides/changing-version.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+page_title: "Upgrading version with the NetApp Ontap Provider"
```
Comment (carchi8py) [inline docs/guides/changing-version.md]: minor

Comment (carchi8py) [reply docs/guides/changing-version.md]: Change the title to have minor as well

File: docs/guides/changing-version.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+subcategory: ""
+description: |-
+---
+
+# Updating version With The NetApp ONTAP Provider
```
Comment (carchi8py) [inline docs/guides/changing-version.md]: minor

**Review by carchi8py — CHANGES_REQUESTED** [review]

**Review by suhasbshekar — COMMENTED** [review]
updated the comments

**Review by carchi8py — APPROVED** [review]

**Pull request merged**